pkg/cloudprovider/aws/fleet/packing/podresources.go

```diff
@@ -0,0 +1,60 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package packing
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type pods []*v1.Pod
+
+func (pods pods) Len() int {
+	return len(pods)
+}
+
+func (pods pods) Swap(i, j int) {
```

Comment:
Is this actually used?

Reply:
Required to satisfy  [sort.Interface](https://golang.org/pkg/sort/#Interface)